enterprise/plugins/com.liferay.ide.server.websphere.core/src/com/liferay/ide/server/websphere/admin/IWebsphereAdminService.java

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
hi greg, I used this service to connect server and get server exactly state and use this to invoke server stop comand. By this way we can exactly know the server status.